### DISCUSSION TITLE
Savannah and github repos are desynchronized

### DISCUSSION THREAD

**Discussion by Boucher-Lambert** [libunwind/libunwind#629] [General]
Hi,

the latest commit on master branch on Savannah repo is [7cf6e84bb86ff5840896b4910ccc3865d4f00ffb](http://git.savannah.nongnu.org/cgit/libunwind.git/commit/?id=7cf6e84bb86ff5840896b4910ccc3865d4f00ffb) dated from 2022-05-24
http://git.savannah.nongnu.org/cgit/libunwind.git/log/

The latest commit on master branch on GitHub repo is https://github.com/libunwind/libunwind/commit/7bcbd0e2e8f8d35cfb3002fb5c199c8464778b90 dated from 2022-09-05
https://github.com/libunwind/libunwind/commits/master

This issue https://github.com/libunwind/libunwind/issues/351 indicates that either the two repos should be synced, or the savannah one be removed.

**Comment by bregma** [comment]
Moving to discussions since this is an issue on `savannah.nongnu.org` and not `github.com/libunwind`.
